docs/source/http_services.rst

```diff
@@ -66,6 +66,9 @@ Individual bokeh plots
 - ``/individual-bandwidth-types``
 - ``/individual-bandwidth-workers``
 - ``/individual-workers-network-bandwidth``
+- ``/individual-workers-network-bandwidth-timeseries``
+- ``/individual-workers-network-cpu-timeseries``
+- ``/individual-workers-network-memory-timeseries``
```

Comment:
Similar comment here about dropping `network-`. We should also add an entry for the disk timeseries plot